Dockerfile

```diff
@@ -28,7 +28,7 @@ ENV OPERATOR=/manager \
     USER_UID=1001 \
     USER_NAME=nonroot
 
-RUN yum -y install shadow-utils
```

Comment:
Please ignore. It won't be merged. This is a workaround for the int-test failures with 
```
#6 0.618 Updating Subscription Management repositories.
#6 0.618 Unable to read consumer identity
#6 0.622
#6 0.622 This system is not registered with an entitlement server. You can use subscription-manager to register.
```